src/AutoRest.CSharp/Common/Generation/Writers/ModelReaderWriterBuildableAttributesWriter.cs

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
```

Comment:
We will need to regenerate all projects by running https://github.com/Azure/autorest.csharp/blob/main/eng/Generate.ps1 and committing all the newly generated files.

Reply:
ModelReaderWriterBuildableAttribute is not an assembly level attribute its a class level attribute that must be applied to the Context class that is written in the library that inherits from ModelReaderWriterContext.

Basically you shouldn't need this class and you should just modify https://github.com/Azure/autorest.csharp/blob/main/src/AutoRest.CSharp/Common/Generation/Writers/ModelReaderWriterContextWriter.cs

Reply:
We also did not run the generate script I mention above since none of the test projects show changes.